### ISSUE TITLE
Can the model provide a confidence for the given result?

### ISSUE THREAD

**Issue by courao** [meijieru/crnn.pytorch#145]
Can the model provide a confidence for the given result?

**Comment by vinayakkailas** [comment]
One way i achieved a score is by, passing the predicted label back to the ctc loss function and taking the anti-log of the negative of the resulting loss

**Comment by Hakan79** [comment]
Hi,

Do you have sample code of getting the confidence score ?

Thanks

**Comment by sangeeth93** [comment]
> One way I achieved a score is by, passing the predicted label back to the CTC loss function and taking the anti-log of the negative of the resulting loss

I hope you would need the ground truth label to find the score using CTC loss. Right?

**Comment by sir-timio** [comment]
Compute confidence score for CTC-decoded text using TensorFlow
https://gist.github.com/githubharald/8b6f3d489fc014b0faccbae8542060dc
